01 - Community/People/ZaherAlMajed.md

```diff
@@ -1,28 +1,28 @@
 ---
 aliases:
-- Zaher Al Majed
+  - Zaher Al Majed
 tags:
-- 
+  -
 publish: true
 ---
 
 # Zaher Al Majed
 
 - GitHub: [ZaherAlMajed](https://github.com/ZaherAlMajed/) ^github
 <!-- - Discord: `@` ^discord-->
-<!-- - Website: <https://> ^website-->
+- Website: <https://zaher.design> ^website
 <!-- - [[Publish sites|Publish site]]: <https://> ^publish-->
 
 %% Feel free to add a bio below this comment %%
 
-
 ## Author of
 
 %% Begin Hub: Released contributions %%
 
 ### Themes
-- [[Illusion]]
-%% End Hub: Released contributions %%
+
+- [[Vicious]]
```

Comment:
Note to self: We would likely introduce an 'Archived themes' heading outside of the Begin Hub/End-hub block, and move the Illusion link to there.